jvm/src/test/scala/webapi/Oas20Test.scala

```diff
@@ -231,27 +231,27 @@ class Oas20Test extends AsyncFunSuite with Matchers with WaitingFileReader {
   test("YAML string parsing with reference and baseUrl param") {
     for {
       unit <- Oas20.parseYaml(
-        apiStringWithRef,
+        apiStringYamlWithRef,
```

Comment:
Regarding this and other `apiStringYamlWithRef` changes: some tests were using incorrect inputs before so I fixed them.